csharp-refactoring/RefactoringChangePreventers/RefactoringChangePreventersTests/PatientTests.cs

```diff
@@ -0,0 +1,35 @@
+using RefactoringChangePreventers.DivergentChange.Correct;
+
+namespace RefactoringChangePreventersTests;
+
+[TestClass]
+public class PatientTests
+{
+    [TestMethod]
+    public void AddTreatment_ShouldAddTreatmentToTreatmentHistory()
```

Comment:
This and all other unit test can we switch to a Given_When_Then naming. Sicne, it's our guideline.